packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2653,8 +2653,64 @@ class SizedOverflowBox extends SingleChildRenderObjectWidget {
 /// painting anything, without making the child available for hit testing, and
 /// without taking any room in the parent.
 ///
+/// Difference in painting can be easily compared by the following sections of with and without Offstage.
+///
+/// ### Without Offstage
+///
+/// ```dart
```

Comment:
Example code should be wrapped in `{@tool dartpad --template=stateless_widget_material}` - see other examples in the code base.

Reply:
It would also be better to have the example just above the "see also" section.

Reply:
Thanks, done

Reply:
@goderbauer  please let me know if any more changes are required